pkg/bucketclaim/bucketclaim.go

```diff
@@ -94,22 +98,28 @@ func (b *bucketClaimListener) Update(ctx context.Context, old, new *v1alpha1.Buc
 }
 
 // Delete processes a bucket for which bucket request is deleted
-func (b *bucketClaimListener) Delete(ctx context.Context, bucketClaim *v1alpha1.BucketClaim) error {
-	klog.V(3).Info("Delete BucketClaim",
+func (b *BucketClaimListener) Delete(ctx context.Context, bucketClaim *v1alpha1.BucketClaim) error {
+	klog.V(3).InfoS("Delete BucketClaim",
 		"name", bucketClaim.ObjectMeta.Name,
 		"ns", bucketClaim.ObjectMeta.Namespace)
 
 	return nil
 }
 
 // provisionBucketClaimOperation attempts to provision a bucket for a given bucketClaim.
+//
+// Recorded events
+//
+//	InvalidBucket - Bucket provided in the BucketClaim does not exist
+//	InvalidBucketClass - BucketClass provided in the BucketClaim does not exist
```

Comment:
I wonder if this is good to doc or not. Seems like this doc could get stale over time if more events are added, or if these events need to change form in any way.